config/locales/en/admin.yml

```diff
@@ -515,9 +516,43 @@ en:
       search:
         title: 'Moderators: User search'
     newsletters:
+      create_success: Newsletter created successfully
+      update_success: Newsletter updated successfully
+      send_success: Newsletter sent successfully
+      delete_success: Newsletter deleted successfully
+      recipients:
```

Comment:
I think we should refactor as well this `newsletters.recipients.*` into `user_segments.*` to make it accesible from other features that will use UserSegments.